Overview.bs

```diff
@@ -3099,9 +3099,49 @@ number of requests:
 
 <h2 id=changes>Changes</h2>
 
-Since the <a href="https://www.w3.org/TR/2024/WD-IFT-20240709/">Working 
+<p id="changes-20250220">Since the <a href="https://www.w3.org/TR/2025/WD-IFT-20250220/">Working 
+  Draft of 20 February 2025</a> (see 
+  <a href="https://github.com/w3c/IFT/commits/main/Overview.bs">commit history</a>):</p>
+
+<ul>
+  <!-- to 20 June 2025 -->
+  <li>Expanded privacy section to provide guidance for encoders.</li>
+  <li>Added examples of full absolute and host name relative URL expansion.</li>
+  <li>Clarify how bytes are read from 'url template bytes'.</li>
+  <li>Added additional error conditions: Op code 0 (insert 0 literals) is made invalid.  Fail if not enough bytes remain for the insert literal operation. Added examples that result in failures.</li>
+  <li>Updated URL expansion examples to use C syntax for the byte arrays. Added more varied examples that include relative URLs and URLs with query parameters.</li>
+  <li>Following TAG review, dropped rfc6570 uri templates, replaced with opcode encoded template.</li>
+  <li>Added invocation of fetch and use the processResponseConsumeBody callback.</li>
+  <li>Added more specific information on configuring the fetch request.</li>
+  <li>Re-worded entry selection in response to review feedback, to be clearer.</li>
+  <li>Specified a selection/fetch ordering for non-invalidating patches.</li>
+  <li>Specified what to do if URL parsing fails.</li>
+  <li>Added a note about optimizing glyph keyed patch application when multiple patches are present.</li>
+  <li>In format 2 interpretation, handling of an unspecified entry delta/length was missing. Added an additional step to generate the URL for that case.</li>
+  <li>Clarified wording related to WOFF2</li>
+  <li>Added a note to the font compression section recommending WOFF2.</li>
+  <li>Updated to (current) shared brotli 15 specification</li>
+  <li>Added flag bits to signal the presence of optional charstring offsets.</li>
+  <li>Added a note about patches adding a CFF table and the charstrings offset.</li>
+  <li>For fonts with CFF or CFF2 tables, added a field to the patch map with offsets to CharStrings.</li>
+  <li>Disallowed undefined variable names in uri template expansion.</li>
+  <li>Reworded statement on erroring for unsupported expression levels.</li>
+  <!-- later undone: Switch back to the name 'URI Templates' to match the reference spec but stick with URL when referring to the output. -->
+  <li>Restricted URL templates to only level one substitutions.</li>
+  <li>Switched from rfc3986 URI to whatwg/url</li>
+  <li>Avoided using 'preload', that term is already used to refer to adding extra data to the initial font.</li>
+  <li>In invalidating patch selection, prioritize entries which have been preloaded thus minimizing round trips.</li>
+  <li>Updated font extension algorithm to use url preload lists.</li>
```

Comment:
This entry is confusing relative to the note at 3132. I believe the terminology @garretrieger settled on is "prefetch lists": https://github.com/w3c/ift-encoder/pull/133#discussion_r2170031245